content/en/docs/reference/config/analysis/ist0108/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: UnknownAnnotation
+layout: analysis-message
+---
+
+This message occurs when you attach an unrecognized annotation in the format `*istio.io[/*]` to a namespace.
```

Comment:
My pseudo regexp `*istio.io[/*]` is too complicated to read, and it applies to more than namespaces.  Just say "This message warns of an undefined `*.istio.io` annotation".

Reply:
fixed